Mathlib/Tactic/Linter/AdmitLinter.lean

```diff
@@ -0,0 +1,62 @@
+/-
+Copyright (c) 2024 Damiano Testa. All rights reserved.
```

Comment:
Should you be the copyright holder? @adomani what do you think?

Reply:
I agree: the copyright holder should probably be Adomas.  I also do not really pay much attention to the copyright section of the files anyway.